modules/custom/az_card/src/Plugin/Field/FieldFormatter/AZCardDefaultFormatter.php

```diff
@@ -125,13 +125,20 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
       // Link.
       $link_render_array = [];
       $link_url = '';
+      $current_path = \Drupal::service('path.current')->getPath();
```

Comment:
With the code change from my other suggestion, we can remove this line since we won't need this variable any longer.

```suggestion
```